cmd/example-hook-sidecar/README.md

```diff
@@ -0,0 +1,37 @@
+# KubeVirt SMBIOS hook sidecar
+
+Example VM definition:
+
+```yaml
+apiVersion: kubevirt.io/v1alpha2
+kind: VirtualMachineInstance
+metadata:
+  creationTimestamp: null
+  labels:
+    special: vm-hook-sidecar-consumer
+  name: vm-hook-sidecar-consumer
+  annotations:
+    # Request the hook sidecar
+    hooks.kubevirt.io/hookSidecars: '[{"image": "registry:5000/kubevirt/example-hook-sidecar:devel"}]'
+    # Overwrite base board manufacturer name
+    smbios.vm.kubevirt.io/baseBoardManufacturer: "Radical Edward"
```

Comment:
I don't follow this line

Reply:
Ah i follow now - the comment should read 

```
The side car container changes the default baseBoardManufacturer in the domxml to this value
```

Reply:
Done